File: 프로그래머스/unrated/181928. 이어 붙인 수/이어 붙인 수.swift
func solution(_ num_list:[Int]) -> Int {
    var sum1 = ""
    var sum2 = ""
    for i in num_list {
        if i % 2 == 0 {
            sum1 += String(i)
        }else{
            sum2 += String(i)
        }
    }
    
    return Int(sum1)! + Int(sum2)!
}

File: 프로그래머스/unrated/181928. 이어 붙인 수/이어 붙인 수.js
function solution(num_list) {
    let hol = ""
    let jjank = ""
    for(let i of num_list) {
        i%2 === 0 ? jjank += String(i) : hol += String(i)
    }
    return Number(hol) + Number(jjank)
}